src/App.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {
+  Card, CardImg, CardText, CardBody, CardLink,
+  CardTitle, CardSubtitle
+} from 'reactstrap';
+import './App.css';
+
+
+
+import {applications} from './shared/applications'
+
+const jobapps = applications;
+
+
+
+
+function Jobs ({item}) {
+  return(
+    <div >
+      <Card className ='jumbotron' >
+          <CardBody>
+            <CardTitle className ='card-title'>{item.company}</CardTitle>
+            <CardSubtitle >{item.type}</CardSubtitle>    
+            <CardText>{item.jobDescriptionUrl}</CardText>
+            <CardText>{item.notes}</CardText>
+            <CardText>{item.createdAt}</CardText>
+            <CardText>{item.updatedAt}</CardText>
+          </CardBody>
+        
+    </Card>
+  </div>
+  )
+
+}
+
+function App() {
+
+  const a = 'PhoneInterviewComplete';
+  const b = 'PhoneInterviewScheduled';
+  const c = 'Applied';
+  const type = {}
+
+  const appListA = jobapps.map(
```

Comment:
Should be able to move all of these into one array method and build in there instead of multiple checking separate conditions.